client/src/appsupport.js

```diff
@@ -20,7 +20,7 @@ function unsupportedBrowser() {
   var isMobile = /Android|BlackBerry|iPhone|iPad|iPod|Opera Mini|IEMobile/.test(
     navigator.userAgent,
   );
-  var isDesktopApp = /DarkLang\/Editor/.test(navigator.userAgent);
+  var isDesktopApp = /DarkLang\/Editor/i.test(navigator.userAgent);
```

Comment:
Instead of doing checks based on user agent (which can be easily changed through DevTools), maybe do progressive enchantment or feature detection instead?

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent for the notes.